bigquery_etl/pytest_plugin/udf.py

```diff
@@ -96,5 +97,8 @@ def runtest(self):
             job_config = bigquery.QueryJobConfig(
                 use_legacy_sql=False, default_dataset=default_dataset
             )
-            job = bq.query(self.query, job_config=job_config)
+            job = bq.query(
+                self.query.replace(GENERIC_DATASET, default_dataset.dataset_id),
```

Comment:
Google has confirmed to me that stored procedures will not support default datasets. To work around that we replace the dataset here.